site/src/api/typesGenerated.ts

```diff
@@ -1020,42 +1020,99 @@ export interface WorkspacesResponse {
 
 // From codersdk/apikey.go
 export type APIKeyScope = "all" | "application_connect"
+export const APIKeyScopes: APIKeyScope[] = ["all", "application_connect"]
 
 // From codersdk/audit.go
 export type AuditAction = "create" | "delete" | "start" | "stop" | "write"
+export const AuditActions: AuditAction[] = [
+  "create",
+  "delete",
+  "start",
+  "stop",
+  "write",
+]
```

Comment:
Surprised to see this type for an array of `AuditActions` generated.

Reply:
This was introduced in this PR so the FE didn't have to maintain a manual list of available enterprise features.